v0.0.2 Improved

Pitometsu opened the pull request:
- Add support for custom title view
- Fix proper behavior for left and right bar
  button items.

Pitometsu committed:
v0.0.2 Improved

- Add support for custom title view
- Fix proper behavior for left and right bar
 button items.

Pitometsu commented:
First of all, thanks for awesome control. It's very helpful.
Could you merge it to master, add 0.0.2 tag and make `pod trunk push` for a new version, please?

I work on https://github.com/Pitometsu/CruiserWebViewController
and have add some general compatibility to your code.

Best regards.

zetachang commented on Classes/IPNavBarSqueezableViewController.m:
```diff
@@ -8,36 +8,51 @@
 typedef NS_ENUM(NSInteger, IPNavBarSqueezingStatus) {
     IPNavBarSqueezingStatusNormal,
+    IPNavBarSqueezingStatusProgress,
     IPNavBarSqueezingStatusSqueezing,
```
Turns out `IPNavBarSqueezingStatusSqueezing` has been replaced by `IPNavBarSqueezingStatusProgress`. Maybe we should removed this status?

Pitometsu replied:
As I understand, it both required: `IPNavBarSqueezingStatusProgress` marks manual slow scrolling at:

``` objc
- (void)squeezeNavBarWithProgress:(CGFloat)delta
```

And `IPNavBarSqueezingStatusSqueezing` still mark complete animation status.

zetachang commented:
Hi, thank you for taking time working on this :+1: 

I've run the example project. And it turns out the squeezed title label is a little bit higher than the original one.

I've pushed some fix to https://github.com/zetachang/IPSqueezableViewController/commit/690af53384f9d96533574bfef5a48dd93497dcf6. It will fix it, but I am not sure if it's still compatible with the `addressField` one you integrate with in your repo.

Hope you can check it out. If it works, we can merge this and release a new version. :wink: 

Regards.

Pitometsu commented:
@zetachang It's ok with custom tittleView, thank you for fast feedback!

Pitometsu committed:
Add customizable title color.

- Oops: remove unnecessary scroll indicator
  insets changing on will appear
- Add scroll view delegate to header.

zetachang commented:
`v0.0.2` has been published. Thank you again for your contributions!

Pitometsu commented:
Thank you for attention!
